server/graphql/schema/mutations/createUser.ts

```diff
@@ -29,8 +33,32 @@ export const createUser = {
     status: { type: GraphQLString },
   },
   async resolve(parent: object, args: object) {
-    const user = await User.create({ ...args, isAdmin: false });
+    try {
+      let tempUser: TempUser = {};
+      let keys = Object.keys(args);
+      let values = Object.values(args);
+
+      // make a temporary user object who creates account
+      for (let i in keys) {
+        tempUser[keys[i]] = values[i];
```

Comment:
why doing this? 
just do 
`const user = { ...args}` and later after password hash just update password? 
user.password = hashedPassword